setup-machine.sh

```diff
@@ -164,21 +164,9 @@ sudo usermod -aG adm vx-services
 # Set up log config
 sudo bash setup-scripts/setup-logging.sh
 
-# Allow mounting of USB
-sudo usermod -aG plugdev vx-ui
-sudo usermod -aG plugdev vx-admin
-
-# set up mount points ahead of time because read-only later
-sudo mkdir -p /media/usb-drive-sda1
-sudo mkdir -p /media/usb-drive-sdb1
-sudo mkdir -p /media/usb-drive-sdc1
-sudo mkdir -p /media/usb-drive-sdd1
-sudo mkdir -p /media/usb-drive-sde1
-sudo mkdir -p /media/usb-drive-sdf1
-sudo mkdir -p /media/usb-drive-sdg1
-sudo mkdir -p /media/usb-drive-sdh1
-
-sudo chown -R vx-ui:vx-group /media/usb-drive*
+# set up mount point ahead of time because read-only later
+sudo mkdir -p /media/vx/usb-drive
+sudo chown -R vx-ui:vx-group /media/vx
```

Comment:
If we want to support multiple USB drives in the future, we could do `/media/vx/usb-drive-sda1, /media/vx/usb-drive-sdb1` and so on. I don't see the need to support labels (which are really only the label of the mount point) or multiple USB drives right now, so simplifying until we have requirements.